leenlab2/reminiscence
# Memory activation trigger

# Description

- Fixed corrupted Pixel font asset
- Refactored PuzzleManager, SceneManager, and MemoryTransition for cleaner logic
- Fixed bug with input manager where OpenTV is still active in memory scene
- Made memory button activate when tape 1 is watched

# How Has This Been Tested?

1. Open tv with no tape. Memory dial should not be active
2. Place tape in tv and watch tape (you can fast forward until about 80% through)
3. Memory dial should go from inactive to active, and trigger an animation
4. You should be able to move around in the memory scene, but not open tv
5. Upon returning, you can use the dial as intended

# Checklists
## Integration Checklist

## Review Checklist
